gamelogic.js

```diff
@@ -18,28 +11,105 @@ function displayWordSoFar(word, guesses) {
 }
 
 function isGameWon(word, guesses) {
-    /*
-    checken of de letter in het woord zit door middel van een for loop
-    inhoud for loop:
-    voor de lengte van het woord een counter opzetten
-    if found counter++
-    if counter === word.length return true
-  */
+    for (let i = 0; i < word.length; i++) {
+        if (guesses.includes(word[i]) && i === word.length - 1) {
+            return true;
+        } else if (!guesses.includes(word[i])) {
+            return false;
+        }
+    }
 }
 
 function isGameLost(word, guesses) {
-    /*
-    checken of de letter in het woord zit door middel van een for loop
-     inhoud for loop:
-    nested loop die per letter checkt of en waar hij in het woord zit
-    voor de lengte van het woord een counter opzetten
-    if found counter++
-    if counter !== word.length && guesses.length is > 7 return true
-  */
+    let counter = 0;
+    for (let i = 0; i < guesses.length; i++) {
+        if (!word.includes(guesses[i])) {
+            counter++;
+        }
+    }
+    return counter >= 7;
+}
+
+function showGallows(word, guesses) {
```

Comment:
Heel goed dat je deze in de gamelogic hebt gezet,  dat had ik ook moeten doen! Veel cleaner!